td-shim/src/bin/td-shim/main.rs

```diff
@@ -149,6 +149,7 @@ pub extern "win64" fn _start(
         pi::fv::SECTION_PE32,
     )
     .expect("Failed to get image file from Firmware Volume");
+    payload = verify_payload(payload, &mut td_event_log);
```

Comment:
Using `#[cfg(feature = "secure-boot")]` may help to simplify implementation of `verify_payload()`. It also help to rename it as `secure_boot_verify_payload()`?

Reply:
Thanks for your suggestions 👍